tethys_cli/manage_commands.py

```diff
@@ -96,5 +105,14 @@ def manage_command(args):
     elif args.command == MANAGE_CREATESUPERUSER:
         primary_process = ['python', manage_path, 'createsuperuser']
 
+    elif args.command == MANAGE_GET_PATH:
+        primary_process = ['python', '-c', f'print("{manage_path}")']
```

Comment:
Seems like overkill to spin up a subprocess to print the `manage_path`. You could just do this:

```
print(manage_path)
```

If you don't set `primary_process` it won't run it.

Reply:
done

Reply:
I got a kick out of this one too.